nbs/models.tft.ipynb

```diff
@@ -15,20 +15,14 @@
    "source": [
```

Comment:
<ul><li>Aligned blocked is not rendering: \begin{align}</li><li>c_{s}, c_{e}, (c_{h}, c_{c}) &amp; = \textrm{GRN}(\textrm{VSN}(\mathbf{E}^{(s)}))</li><li>\end{align}</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>